src/ecrc-frontend/src/components/page/userConfirmation/UserConfirmation.js

```diff
@@ -61,9 +61,26 @@ export default function UserConfirmation({ page: { header, setApplicant } }) {
         const birthDt = birthdate.split("-").join("/");
 
         // Convert province name
-        // TODO: Implement properly for all provinces
-        const provinceNm =
-          region === "BC" ? "BRITISH COLUMBIA" : "Fail Province";
+        const regionMap = new Map([
+          ["BC", "BRITISH COLUMBIA"],
+          ["AB", "ALBERTA"],
+          ["NL", "NEWFOUNDLAND AND LABRADOR"],
+          ["PE", "PRINCE EDWARD ISLAND"],
+          ["NS", "NOVA SCOTIA"],
+          ["NB", "NEW BRUNSWICK"],
+          ["QC", "QUEBEC"],
+          ["ON", "ONTARIO"],
+          ["MB", "MANITOBA"],
+          ["SK", "SASKATCHEWAN"],
+          ["YT", "YUKON"],
+          ["NT", "NORTHWEST TERRITORIES"],
```

Comment:
To match what we get from the backend API, can we change:
NORTHWEST TERRITORIES to NORTH WEST TERRITORIES
NEWFOUNDLAND AND LABRADOR to NEWFOUNDLAND

Reply:
Done.